assets/header.txt

```diff
@@ -34,6 +34,7 @@
 ##INFO=<ID=CIPOS,Number=2,Type=Integer,Description="PE confidence interval around POS">
 ##INFO=<ID=CNV_RATIO,Number=1,Type=Float,Description="Copy number ratio">
 ##INFO=<ID=CHR2,Number=1,Type=String,Description="Chromosome for POS2 coordinate in case of an inter-chromosomal translocation">
+##INFO=<ID=CSQ,Number=1,Type=String,Description="The consequence of the structural variant">
```

Comment:
maybe also mention this comes from ensembl vep

Reply:
Done! :p